paddle/fluid/framework/ir/mkldnn/cpu_quantize_pass.cc

```diff
@@ -934,6 +926,11 @@ void CPUQuantizePass::QuantizeMatmul(Graph* graph, bool with_residual) const {
                           is_x_unsigned,
                           is_y_unsigned));
 
+    if (matmul_op->Op()->Type() == "matmul" ||
+        matmul_op->Op()->Type() == "matmul_v2") {
+      matmul_op->Op()->SetType("fused_matmul");
+    }
```

Comment:
I think we could move this code to _cpu_quantize_placement_pass_, because `QuantizeMatmul` has extra checks which allow quantization only if operator has `mkldnn_data_type`=="int8". We are going to remove all oneDNN-specific attributes from base matmul operator, so these attributes won't be accessible.

Reply:
Okay, I'll move it to cpu_quantize_placement_pass. And since cpu_quantize_placement is always before cpu_quantize_pass, it can be inferred that only `fused_matmul` should be searched in cpu_quantize_pass.